assets/js/zzzz-search-data.json

```diff
@@ -0,0 +1,58 @@
+---
+permalink: /assets/js/search-data.json
```

Comment:
is this some generated file.. i am not able to review :) please add some writeup.

Reply:
Yeah , this slipped through .. I have removed this , 
Sorry for the confusion